core/jazz_codeq/utils.js

```diff
@@ -57,7 +57,7 @@ function getReport(metrics, sonarMeasures, config) {
 					let metricName = Object.keys(config.METRIC_MAP).find(key => config.METRIC_MAP[key] === record.metric);
 					output.metrics.push({
 						"name": metricName,
-						"link": config.SERVICE_API_URL + config.HELP_SERVICE + "?metrics=" + metricName,
+						"link": config.SONAR_PROTOCOL + config.SONAR_HOSTNAME + '/component_measures?id=jazz_' + serviceContext.query.domain + '_' + serviceContext.query.service + '_' + branch + '&metric='+metricName,
```

Comment:
i dont think this should say jazz_.  I believe this is the project key and should come from config